mod_ci/controllers.py

```diff
@@ -763,7 +781,48 @@ def progress_reporter(test_id, token):
     return "FAIL"
 
 
+@mod_ci.route('/show_maintenance')
+@template_renderer('auth/maintenance.html', 404)
```

Comment:
maintenance is controlled by admin and admin is a part of authentication. Thats why i was confused where to place it. i will place it outside of auth.